src/index.tsx

```diff
@@ -1,14 +1,39 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  HashRouter, Navigate, Route, Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { NotFoundPage } from './components/NotFoundPage/NotFoundPage';
 import { App } from './App';
 
 createRoot(document.getElementById('root') as HTMLDivElement)
   .render(
-    <Router>
-      <App />
-    </Router>,
+    <HashRouter>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+
+          <Route path="/home" element={<Navigate to="/" replace />} />
+          <Route path="people">
+            <Route
+              index
+              element={(<PeoplePage />)}
```

Comment:
You don't need the round brackets here
```suggestion
              element={<PeoplePage />}
```